orga/.eslintrc.js

```diff
@@ -11,7 +13,7 @@ module.exports = {
     'ember'
   ],
   extends: [
-    'eslint:recommended',
+    ...(fs.existsSync('../.eslintrc') ? ['../.eslintrc'] : []),
```

Comment:
A voir

Reply:
Je soupçonne que cette déclaration (et les autres identiques) soit superflue. D'après la doc de ESLint (https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy) c'est déjà le comportement par défaut de ESLint.

Reply:
Avec cette déclaration les règles de lint du root sont appliquées, sans non. Si eslint trouve un eslintrc il s'arete là. A savoir que le lint se fait depuis le sous-dossier (mon-pix, orga, ...) donc eslint n'a aucune raison de remonter. Si le lint se faisait à partir du directory root alors pas besoin de cette ligne.

Reply:
Plutôt que passer par un `[]` qu'on décompose avec `...`, pas possible de faire juste ça ?
`fs.existsSync('../.eslintrc') ? '../.eslintrc' : ''`

Reply:
Avec le destructuring d'un array vide l'array "extends" n'obtiens pas d'item supplémentaire. 
Avec `''` l'array obtient un item supplémentaire qui cause une erreur.

Reply:
Et en enlevant le `root: true` de ce `.eslintrc` ? Je fais peut-être trop confiance à la doc mais bon 🤷🏼‍♂️

Reply:
Et en enlevant le `root: true` de ce `.eslintrc` ? Je fais peut-être trop confiance à la doc mais bon 🤷🏼‍♂️

Reply:
Oui cela semble être une bonne solution.

Reply:
Erf il faut quand même cette ligne pour que eslint puisse voir le eslintrc parent, même en enlevant `root: true`.

Reply:
Par contre je challenge l'utilité du fs.existSync

Reply:
J'ai enlevé root: true là où il était inutile. Je rajoute des "A voir".